src/MUIDataTable.js

```diff
@@ -368,7 +368,7 @@ class MUIDataTable extends React.Component {
       columnData.push(columnOptions);
 
       filterData[colIndex] = [];
-      filterList[colIndex] = [];
+      filterList[colIndex] = columnOptions.filterList || [];
```

Comment:
buildColumns should return filterList based on the initial columnOptions filterList